src/styles/components/MasterHeader.scss

```diff
@@ -53,6 +54,16 @@ nav.navbar{
   }
   .btn-link{
     height: 40px;
+    padding-left: 0;
+    padding-right: 0;
+    width: 100%;
+    text-align: left;
+
+    .caret {
+      position: absolute;
+      right: 12px;
+      top: 19px;
```

Comment:
we can delete this PR and I'll do more experimental CSS locally eventually
inshaallah, and make sure it's compatible, if I come up with anything I'll
PR again

simple is good

On Sun, Apr 10, 2016, 12:35 AM Mohamed El Mahallawy <
notifications@github.com> wrote:

> In src/styles/components/MasterHeader.scss
> <https://github.com/quran/quran.com-frontend/pull/258#discussion_r59122780>
> :
>
> > @@ -53,6 +54,16 @@ nav.navbar{
> >    }
> >    .btn-link{
> >      height: 40px;
> > +    padding-left: 0;
> > +    padding-right: 0;
> > +    width: 100%;
> > +    text-align: left;
> > +
> > +    .caret {
> > +      position: absolute;
> > +      right: 12px;
> > +      top: 19px;
>
> not sure but I know flex is tricky and not widely supported
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/quran/quran.com-frontend/pull/258/files/e00db328e72a70a6f672982312e611272f2e6d0a#r59122780>
>